src/layers/conv.jl

```diff
@@ -1065,4 +1065,102 @@ function (l::MEGNetConv)(g::GNNGraph, x::AbstractMatrix, e::AbstractMatrix)
     return x̄, ē
 end
 
+@doc raw"""
+    GMMConv(in => out, K, e_dim, σ=identity; [init, bias])
+Graph mixture model convolution layer from the paper [Geometric deep learning on graphs and manifolds using mixture model CNNs](https://arxiv.org/abs/1611.08402)
```

Comment:
```suggestion
    GMMConv(in => out, e_dim, σ=identity; [K, init, bias])

Graph mixture model convolution layer from the paper [Geometric deep learning on graphs and manifolds using mixture model CNNs](https://arxiv.org/abs/1611.08402).
```

Reply:
Maybe the best compromise is to have `e_dim` has a mandatory positional argument and `K`  as keyword argument with default 1. Or we could have `e_dim` *mandatory* keyword arg. What do you think?  I don't want both of them to be positional since something like `GMMConv(16 => 16, 3, 5)`  is very unclear.

Reply:
Actually, we should adopt the interface provided by `CGConv`  and have `GMMConv((in, ein) => out, act=identity; K=1, bias=true, init=glorot_uniform, residual=false)`.

Reply:
Working on it.

Reply:
Added in the [commit](https://github.com/melioristic/GraphNeuralNetworks.jl/commit/510e4309b36e064ac42c441429e97b6990313d1d)